### ISSUE TITLE
[Must Have] Implement review option

### ISSUE THREAD

**Issue by overeha** [code4romania/war-support-un-acoperis#151]
The users must be able to add a review to their host, respectively their guest. 
This review should be displayed on their profile for the admin to be able to see if there are any issues with any of the hosts or refugees. 
The review should be: 
- a 1 to 5 stars
- a message box.

**Comment by overeha** [comment]
Every Refugee and every Host has a profile and an account. Once a Refugee has been allocated to a Host then, if the Refugee did stay at that host they should be able to go into their account, click on the Hosts profile (currently a Refugee can see in his Admin the Host he is allocated to) and he should be able to post a review at the bottom of the page in a simple box. 

The Refugee profile is accessible, after log in, by clicking on the username in the main menu and selecting my profile. 
In his profile, the Refugee can see his requests, the status of his requests and the host he has been allocated to and the host profile. 

I am afraid I cannot provide dummy data for users right now as I do not have access to that and the main dev team is currently unavailable. If this is a blockage, I could generate data tomorrow and we can postpone this feature for a later date.

**Comment by sherbastian** [comment]
i'll take it

**Comment by sherbastian** [comment]
There's still some fine tuning on the admin side:
![image](https://user-images.githubusercontent.com/20094749/156861519-81bd52cf-9af3-492b-bbc2-9e1788032c12.png)

The user's page before leaving a comment:
![image](https://user-images.githubusercontent.com/20094749/156861735-142cde0f-c599-41c0-977b-b2f3856f7eac.png)

The user's page after leaving a review:
![image](https://user-images.githubusercontent.com/20094749/156861648-e59d26d5-5098-417d-b771-7ccc878389f3.png)

**Comment by sherbastian** [comment]
https://github.com/code4romania/war-support-un-acoperis/pull/165 solves the part where refugees get to review their accommodation but the inverse should be implemented as well where hosts get to review their guests. 
This is not possible yet due to the fact that hosts don't have a view where they can see their guests. Once that view is made available the rating/review system can be easily extended.

**Comment by overeha** [comment]
@gheorghelupu17 is this implemented already? can we test and then push to production if it works. Please indicate if this is merged so we can properly test it.

**Comment by gheorghelupu17** [comment]
> @gheorghelupu17 is this implemented already? can we test and then push to production if it works. Please indicate if this is merged so we can properly test it.

No, we need to make a rebase and retest @sherbastian  can you manage this?
